README.md

```diff
@@ -44,10 +44,18 @@ Read more about this in the [Lumberjack](#Lumberjack) API section.
 ```javascript
 import Lumberjack from '@americanexpress/lumberjack';
 
+let formatter = null;
```

Comment:
What's the goal of moving formatter out of the Lumberjack config object into it's own variable?

Reply:
Valid inquiry - the main reason was to avoid the duplicate key. I've updated the example to remove the `let` in 9e1623c.